.rubocop.yml

```diff
@@ -3,16 +3,19 @@ inherit_mode:
     - Exclude
 
 require:
+  - standard
   - rubocop-performance
   - rubocop-rails
   - rubocop-rspec
-  - standard
 
 inherit_gem:
   standard: config/base.yml
+  standard-performance: config/base.yml
+  standard-custom: config/base.yml
```

Comment:
so were we not using these before?

Reply:
We were using the `standard-performance` rules, but the recommended way to include them changed. `standard-custom` is a new thing that we get for free with the `standard` gem, but requires opting in via config. I think it's literally one new rule. This change is keeping up with the latest prescribed practices.